tools/testrunner/local/statusfile.py

```diff
@@ -63,7 +63,7 @@
 for var in ["debug", "release", "big", "little", "android",
             "arm", "arm64", "ia32", "mips", "mipsel", "mips64", "mips64el",
             "x64", "ppc", "ppc64", "s390", "s390x", "macos", "windows",
-            "linux", "aix", "r1", "r2", "r3", "r5", "r6", "riscv", "riscv64"]:
+            "linux", "aix", "r1", "r2", "r3", "r5", "r6", "riscv","riscv64"]:
```

Comment:
`"riscv"` is still in this list.

Reply:
delete it needs delete 'riscv' from *.status

Reply:
I had do it